tests/st/calicoctl/test_upgrade.py

```diff
@@ -374,7 +393,7 @@
                  'name': 'eth0',
                  'node': 'rack1-host1',
                  'orchestrator': 'k8s',
-                 'workload': 'default/frontend-5gs43'},
```

Comment:
`/` is a valid character in workload names in 2.6.X.  Why are you changing it here?